WalletWasabi.Fluent/App.axaml.cs

```diff
@@ -44,6 +46,21 @@ public App(Func<Task> backendInitialiseAsync) : this()
 		public override void Initialize()
 		{
 			AvaloniaXamlLoader.Load(this);
+			Services.SingleInstanceChecker.OtherInstanceStarted += (sender, args) =>
+			{
+				Dispatcher.UIThread.Post(() =>
+				{
+					if (ApplicationLifetime is IClassicDesktopStyleApplicationLifetime desktop)
```

Comment:
Should this condition possibly be moved outside to L49? I mean it does not seem to be useful to do any work to find out here that it was for nothing.